packages/nouns-webapp/src/components/Ideas/index.tsx

```diff
@@ -0,0 +1,196 @@
+import { Alert, Button } from 'react-bootstrap';
+import classes from './Ideas.module.css';
+import { useEthers } from '@usedapp/core';
+import { isMobileScreen } from '../../utils/isMobile';
+import clsx from 'clsx';
+import { useUserVotes } from '../../wrappers/nounToken';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+import { useEffect, useState } from 'react';
+import { SiweMessage } from 'siwe';
+
+const HOST = "http://localhost:5001";
```

Comment:
prob check which env this is running in to use local or prod

Reply:
Yeah just left this here for now while working in dev. Not sure what the prod endpoint even is.